run-videochat.sh

```diff
@@ -84,6 +84,7 @@ check_os_version() {
 
 error() {
     zenity --error --no-wrap --text "$@" > /dev/null 2>&1
+    adb forward --remove tcp:$PORT
```

Comment:
You should change the error message about the port being in use to a yes/no dialog asking whether we should attempt to free up the port by killing ADB. You should not be changing this error function, which is only a thin wrapper over Zenity and is used elsewhere in the script.

Reply:
port error has been reimplemented as its own function with a question dialog. it'll now ask to clear the port on startup if busy rather than indiscriminately on exit